modules/es7.promise.finally.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+// https://github.com/tc39/proposal-promise-finally
+var $export = require('./_export');
+
+$export($export.P, 'Promise', {
+  'finally': function _finally(onFinally){
+    var handler = typeof onFinally === 'function' ? onFinally : function(){};
+    var C = Promise;
+    var newPromise = Promise.prototype.then.call(
+      this, // throw if IsPromise(this) is not true
+      function(x) {
+        return new C()(function(resolve) {
```

Comment:
That's the behavior of the polyfill:
https://github.com/tc39/proposal-promise-finally/blob/84cec8005364a0cae413cad3be1e47b6a26be369/polyfill.js#L33